### ISSUE TITLE
Wrong UV values

### ISSUE THREAD

**Issue by MathisEngels** [pmndrs/gltfjsx#236]
Hi.

When I use gltfjsx, the UV of an object gets replaced with wrong values.
For example : 
Original.glb file : 
![image](https://github.com/pmndrs/gltfjsx/assets/1526270/154ddd3b-d7be-4381-8982-5e8be062f94e)

As you can see on the above picture, Table as UV and the first 3 are : 0.4732959568500519, 0.8645936250686646, 0.2926514446735382

Now I run `npx gltfjsx@6.2.13 .\scene1-rigid.glb -T -f png -k -j` on the file.
This is what I get : 
![image](https://github.com/pmndrs/gltfjsx/assets/1526270/773dceb8-379b-4005-870e-2a0cd68e0a3e)

UV for "Table001" are changing for 0.0625 (for the 152 values).
_0.0625 never appear in the original UV array._

_Note: If you wonder why instanceUVs and instanceNames arrays goes from 30 to 10 in size, it's because there is instancedMeshes._

TLDR : UV array after using gltfjsx gets changed to an array filled with a random value.

Files : 
[scene1.zip](https://github.com/pmndrs/gltfjsx/files/13529585/scene1.zip)

**Comment by donmccurdy** [comment]
Depending on the configuration of the optimization pipeline, glTF Transform may reorder vertices, eliminate some vertices, or change the numerical values of vertex attributes. The intention of the optimizations is that the scene should look the same, while being represented in a more compact way. If the model does not look visually the same, that could be a bug we could investigate. If you require that no vertex data change in any way, then little optimization is possible, and you probably need to skip the `-T` flag.
